### PR TITLE
[IMP] event_project: As followers of the event, followers and users w…

### PR DESCRIPTION
**Pull request by alfredoavanzosc**
…ill get from project.

### CODE REVIEW

**Comment by alfredoavanzosc** [comment]
Buenos días @pedrobaeza . He modificado el módulo "event_project" para meterle las mismas modificaciones que "project_events" de avanzosc. También he modificado el módulo "partner_event"

**Comment by alfredoavanzosc** [comment]
[IMP] partner_event: Get as followers of the event, people registered in the event.

File: event_project/__openerp__.py [hunk 1]
```diff
@@ -8,6 +7,7 @@
               "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
               "Odoo Community Association (OCA)",
     "website": "http://www.antiun.com",
+    "images": [],
```
Comment (pedrobaeza) [inline event_project/__openerp__.py:10]: This is intended to content some images, but not putting empty to bypass the warning.

**Comment by pedrobaeza** [comment]
Alfredo, you're modifying 2 modules in this PR. Is this intended?

@antespi, what do you think about the change?

**Comment by alfredoavanzosc** [comment]
Hello @pedrobaeza  and @antespi .

I extended 2 modules:
1.- event_project: When a project is assigned to an event, followers and people team of the project, automatically are enlisted as followers of the event.
2.- partner_event: In de event, make followers to each partner who is enlisted in the list of registered.

I put the 2 modifications in one PR, because if I put them separately, the PR of the "partner_event" module, gave an error of "coveralls" caused by the module " "event_project".

Regards

I forgot to mention, if not put the key "images", he gave an error.

**Pull request closed without merging**